src/css/components/main.css

```diff
@@ -63,6 +74,19 @@
   font-weight: bold;
 }
 
+.main-title {
+  font-size: 27px;
+  padding: 30px;
+}
+
+.main-or {
+  padding: 30px 0;
+}
+
+.create-new-list-btn {
+  margin: 0px 50px;
```

Comment:
Wherever you have something with a value of `0` you can leave off the unit. 0 of anything is 0! So, this can change to `margin: 0 50px;`

Reply:
you're right, thanks for this!